Formula/php.rb

```diff
@@ -0,0 +1,260 @@
+    sbin.install "sapi/fpm/init.d.php-fpm" => "php#{version.to_s[0..2].delete(".")}-fpm"
+
+    if !(config_path/"php-fpm.d/www.conf").exist? && (config_path/"php-fpm.d/www.conf.default").exist?
+      mv config_path/"php-fpm.d/www.conf.default", config_path/"php-fpm.d/www.conf"
+    end
+
+    unless (config_path/"php-fpm.conf").exist?
+      config_path.install "sapi/fpm/php-fpm.conf"
+      inreplace config_path/"php-fpm.conf" do |s|
+        s.sub!(/^;?daemonize\s*=.+$/, "daemonize = no")
+      end
+    end
+  end
+
+  def caveats
+    s = []
+
+    s << <<-EOS.undent
+      To enable PHP in Apache add the following to httpd.conf and restart Apache:
+          LoadModule php7_module #{HOMEBREW_PREFIX}/opt/php#{version.to_s[0..2].delete(".")}/libexec/apache2/libphp7.so
+
+          <FilesMatch \.php$>
+              SetHandler application/x-httpd-php
+          </FilesMatch>
+
+      Finally, check DirectoryIndex includes index.php
+          DirectoryIndex index.php index.html
+    EOS
+
+    s << <<-EOS.undent
+      The php.ini file can be found in:
+          #{etc}/php/#{php_version}/php.ini
+    EOS
+
+    s.join "\n"
+  end
+
+  def post_install
+    (var/"log").mkpath
+    touch var/"log/php-fpm.log"
+
+    chmod 0755, lib/"php/.channels"
+    chmod 0755, lib/"php/.channels/.alias"
+    chmod 0644, (Dir.glob(lib/"php/.channels/**/*", File::FNM_DOTMATCH).reject { |a| a =~ %r{\/\.{1,2}$} || File.directory?(a) })
+
+    %w[
+      php/.depdblock
+      php/.filemap
+      php/.depdb
+      php/.lock
+    ].each do |f|
+      chmod 0644, lib/f
+    end
+
+    php_ini = "#{etc}/php/#{php_version}/php.ini"
+    system bin/"pear", "config-set", "php_ini", php_ini, "system"
+
+    %w[
+      ldap
+      mcrypt
+      tidy
+      imap
+      opcache
+    ].each do |e|
+      config_path = (etc/"php/#{php_version}/conf.d/ext-#{e}.ini")
+      extension_type = (e == "opcache") ? "zend_extension" : "extension"
+      if config_path.exist?
+        inreplace config_path, %r{#{extension_type}=.*$}, "#{extension_type}=#{e}.so"
+      else
+        config_path.write <<-EOS.undent
+          [#{e}]
+          #{extension_type}="#{e}.so"
+        EOS
+      end
+    end
+  end
+
+  def php_version
+    version.to_s.split(".")[0..1].join(".")
+  end
+
+  plist_options :startup => true, :manual => "php-fpm --nodaemonize --fpm-config #{HOMEBREW_PREFIX}/etc/php/#{version.to_s.split(".")[0..1].join(".")}/php-fpm.conf"
+
+  def plist; <<-EOPLIST.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <true/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_sbin}/php-fpm</string>
+          <string>--nodaemonize</string>
+          <string>--fpm-config</string>
```

Comment:
Same as above, no need for this option. `--nodaemonize` should propbably stay here as that's what `launchd` is expecting.